tests/phpunit/elementor/modules/dev-tools/test-deprecation.php

```diff
@@ -226,8 +226,12 @@ public function test_do_deprecated_action__soft() {
 			echo 'Testing Do Deprecated Action';
 		} );
 
+		ob_start();
```

Comment:
maybe add a comment? (well, actually the next dev will see the commit message, so I think it's ok?)

Reply:
Why comment?